### ISSUE TITLE
The buttons override each other in supervan theme

### ISSUE THREAD

**Issue by eladcandroid** [craftpip/jquery-confirm#58]
![1](https://cloud.githubusercontent.com/assets/13297000/11971983/393038b6-a94e-11e5-9123-560b2c098288.png)

With code:
```javascript
    $.confirm({
        title: "הכניסו את כתובת האימייל",
        confirmButton: "המשיכו בתהליך הרישום",
        cancelButton: "ביטול",            
        rtl: true,
        theme: "supervan",
        content: "<div class=\"form-group\">"+
            "<input autofocus type=\"text\" id=\"input-name\" placeholder=\"כתובת אימייל\" class=\"form-control\">"+
        "</div>"+
        "<p class=\"text-danger\" style=\"display:none\">לא הוכנסה כתובת אימייל</p>",
        confirm: function () {
            var input = this.$b.find("input#input-name");
            var errorText = this.$b.find(".text-danger");
            if (input.val() == "") {
                errorText.show();
                return false;
            } else {
                alert("Hello " + input.val());
            }


        }
    });
```

**Comment by eladcandroid** [comment]
OK, I found the fix. I need to add something like:

columnClass: "span6 offset3 col-md-8 col-md-offset-2",

to the parameters for making the box bigger

**Comment by craftpip** [comment]
Hello @eladcandroid , I'm working on the new themes, will soon publish it. 

Thanks ! :+1:

**Comment by craftpip** [comment]
This is fixed in v2.5 where in supervan the buttons do not have fixed widths.
